stackstorm.yml

```diff
@@ -6,11 +6,13 @@
     - StackStorm.mongodb
     - StackStorm.rabbitmq
     - role: StackStorm.postgresql
-      when: not (ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8')
+      when: not (ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8') and
+            not (ansible_facts.distribution == 'Ubuntu' and ansible_facts.distribution_major_version == '18')
     - StackStorm.st2repo
     - StackStorm.st2
     - role: StackStorm.st2mistral
-      when: not (ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8')
+      when: not (ansible_facts.os_family == 'RedHat' and ansible_facts.distribution_major_version == '8') and
+            not (ansible_facts.distribution == 'Ubuntu' and ansible_facts.distribution_major_version == '18')
```

Comment:
I did consider that but decided against it as Debian has it's own release versions and release cycle unlike the way that CentOS and RHEL are in step.  
I'm happy to make the change to Debian though if that is the convention.

Reply:
Is os-distribution used in other places in the repo? (Not in position to search at moment). Armab probably best to comment if preference for one or other.

Reply:
There was a need to verify `ansible_facts.os_family == RedHat` because that includes both CentOS and RedHat systems.
`ansible_facts.distribution` in this particular case looks fine, considering we need to check for Ubuntu Bionic version.

Anyways, st2mistral will be removed later as part of the [Mistral Deprecation Plan](https://github.com/StackStorm/st2/issues/4762).